### DISCUSSION TITLE
Problem with compiling PythonAPi

### DISCUSSION THREAD

**Discussion by yeetnas** [carla-simulator/carla#6495] [Q&A]
hey, 
my problem is that whenever i try to input the command " make PythonAPI "the answer is that the command is unknown. I'm in the root folder of carla and in the correct command window. 
Can anyone help me with that?

**Comment by jcastillo-lazr** [comment]
I guess depending on your OS, have you tried?

On Ubuntu for example: `sudo apt install make` or do you have any build-essential package installed?

On Windows, you need to install make and add it to your PATH
